explanations/machine-learning-terms.md

```diff
@@ -49,6 +49,27 @@ Researchers often invent complicated names for simple things. This page will hel
     <p>Technically, a kernel function allows making complex non-linear classifiers using any learning algorithm that can be expressed solely in terms of dot products between two vectors.</p>
 </td>
 </tr>
+<tr>
+<td valign="top">Hyperparameters</td>
+<td>
+    <p>They indicate what the model looks like and how it will be trained. For example, in the case of neural networks:</p>
```

Comment:
I would suggest reformulating this a bit.

"Hyperparameters specify the model's architecture and how it will be trained."

Reply:
I would also add that these parameters are not learned during training.